ganga/GangaCore/Core/GangaRepository/GangaRepositoryDatabase.py

```diff
@@ -208,12 +208,16 @@ def start_database(self):
         HOST = self.database_config["host"]
         connection_string = f"mongodb://{HOST}:{PORT}/"
         client = pymongo.MongoClient(
-        connection_string, serverSelectionTimeoutMS=10000)
+            connection_string, serverSelectionTimeoutMS=10000)
         self.connection = client[self.db_name]
 
         self.container_controller = controller_map[self.database_config["controller"]]
-        self.container_controller(
+        start_flag, error = self.container_controller(
             database_config=self.database_config, action="start", gangadir=self.gangadir)
+        if start_flag is False and error is not None:
+            # quit ganga process as the first registry could not start the database container
+            import sys
+            sys.exit()
```

Comment:
Is there a better way to quit ganga? Error during startup would imply that the first registry faced this error and thus I assume simply quitting everything would be fine.

Reply:
IIRC a
 ```
logger.fatal
```
will cause a shutdown after the message.